### ISSUE TITLE
not a streaming model

### ISSUE THREAD

**Issue by hm-0714** [tencent-ailab/3m-asr#1]
i doubt that  this model  is not a streaming model , is that right？

**Comment by ucasyouzhao1987** [comment]
Yeah, but you can make a simple configuration to make a streaming model.

**Comment by ododoyo** [comment]
Yes, our experiments and the given recipe is for a full-chunk model. But the code retains the method of U2 training in WeNet for a streaming model, you can set `use_dynamic_chunk=True`  to have a try.

**Comment by hm-0714** [comment]
OK ,thank you for your reply
